src/bridge/nid_util/nid_util_test.cc

```diff
@@ -38,7 +38,7 @@ struct CorrespondingNid {
   DigestCase expected;
 };
 
-constexpr CorrespondingNid kNidMapping[] = {
+const CorrespondingNid kNidMapping[] = {
```

Comment:
Just curious, what was the difference?

Reply:
Can't be `constexpr` here because it's calling some OpenSSL `EVP_*` functions (fails to compile otherwise). I think this slipped by when I was renaming the "Corresponding" structs in the test cases so the variable name was separate from the struct definition.